dll/win32/msgina/shutdown.c

```diff
@@ -238,6 +272,89 @@ ForceFriendlyUI(VOID)
     return FALSE;
 }
 
+BOOL
+DrawIconOnOwnerDrawButtons(
+    DRAWITEMSTRUCT* pdis,
+    HINSTANCE hInst)
+{
+    HICON hCurrIcon;
+    RECT rect;
+    
+    rect = pdis->rcItem;
+
+    /* Check the button ID for revelant icon to be used */
+    switch (pdis->CtlID)
+    {
+        case IDC_BUTTON_SHUTDOWN:
+        {
+            if (pdis->itemState & ODS_SELECTED)
+            {
+                hCurrIcon = LoadImageW(hInst, MAKEINTRESOURCE(IDI_BUTTON_SHUTDOWN_PRESSED), IMAGE_ICON, 0, 0, LR_DEFAULTCOLOR);
```

Comment:
Since you're explicitly using Unicode variant of the functions from the API, use `MAKEINTRESOURCEW` as well.